lib/src/sdk_env.dart

```diff
@@ -69,13 +62,49 @@ class DartSdk {
   }
 }
 
+class FlutterSdk {
+  String _flutterBin;
+
+  FlutterSdk({String sdkDir}) {
+    String path = _prepareSdkPath(sdkDir);
+    _flutterBin = '${path}flutter';
```

Comment:
Why not p.join w/ the path package?

I avoid path concat w/ strings like the plague